modules/understanding-upgrade-channels.adoc

```diff
@@ -7,104 +7,97 @@
-endif::openshift-origin[]
 
-ifdef::openshift-origin[]
-The presence of an update recommendation in the `stable-4` channel at any point is a declaration that the update is supported. While releases will never be removed from the channel, update recommendations that exhibit serious issues will be removed or made conditional from all channels. When an update recommendation is supported, it remains supported for the life of {product-version}, even if the update recommendation is later dropped or made conditional.
-endif::openshift-origin[]
 
 ifndef::openshift-origin[]
 
 [id="fast-stable-channel-strategies_{context}"]
-== Fast and stable channel use and strategies
+== Choosing the correct channel for your cluster
+
+Because update recommendations available in both `fast-{product-version}` and `stable-{product-version}` are fully supported and benefit equally from extensive data analysis the choice between the two channels should be guided based on your specific needs rather than which channel is the default or qualities implied by the channel names. Largely this is a choice between updating to the latest GA versions as soon as Red Hat is willing to offer support or waiting while Red Hat collects additional data before promoting to the stable channel.
+
+Additionally, there are several factors which may lead an organization to move clusters to the fast channel either permanently or temporarilly including:
+ - The desire to apply a specific fix known to affect your environment without delay.
+ - Application of CVE fixes without delay -- Because even CVE fixes may introduce regressions we still apply promotion delays to z-streams with CVE fixes.
+ - Internal pre-prod promotion processes -- If it takes your organization several weeks to promote updates between environments it would be best to use the latest available rather than introducing additional weeks or months of delay.
+
+=== Minor version initial update rollouts
+
+Because the scope of changes associated with a z-stream update is orders of magnitude lower than the scope of changes between any two minor versions our data collection requirements are similarly orders of magnitude different as well. While we can safely promote z-stream updates within a week or two of GA we cannot do the same with minor version updates. Therefore, it's expected that the initial availability of update recommendations to the latest minor may take 45-90 days to appear in the stable channel.
+
+Given this lengthy data collection phase it's important that any organization which has their own lengthy production qualification processes begin testing at GA rather than waiting for stable channel update recommendations to become available. Waiting for stable channel update edges only further delays defect resolution of any defect unique to your usecase or environment.
```

Comment:
We should mention fast channel if we are mentioning stable channel.  Suggested text `any organization which has their own lengthy production qualification processes begin testing using fast channel at GA rather than waiting for stable channel`

Reply:
I got rid of this paragraph in a refactoring and I think this paragraph covers your suggestion now. https://github.com/openshift/openshift-docs/pull/47333/files#diff-a730a4f4ac2c41dde208df2277ff16a4411c015a889dc4ba15323ed8805df5ceR91-R94